evm/utils/db.py

```diff
@@ -8,42 +8,50 @@
     EMPTY_SHA3,
 )
 
+from evm.rlp.blocks import BaseBlock
+from evm.rlp.headers import BlockHeader
 
-def make_block_number_to_hash_lookup_key(block_number):
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from evm.db.chain import BaseChainDB  # noqa
+
+
+def make_block_number_to_hash_lookup_key(block_number: int) -> bytes:
     number_to_hash_key = b'block-number-to-hash:%d' % block_number
     return number_to_hash_key
 
 
-def make_block_hash_to_score_lookup_key(block_hash):
+def make_block_hash_to_score_lookup_key(block_hash: str) -> bytes:
```

Comment:
Why `str` and not `bytes`?

Reply:
Yeah, good point. I typed that file before I figured out how to get monkeytype to help me (which I started to use later) and it seemed natural to have these lookup methods take `str`. If I had run this file through monkeytype (which I now did just to double check) I would have noticed that these methods take bytes.

I guess as a rule of thumb we are always dealing with bytes instead of strings (which makes a lot of sense so we don't have to worry about encodings)